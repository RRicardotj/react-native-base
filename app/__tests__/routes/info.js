import 'react-native';
import React from 'react'; // eslint-disable-line no-unused-vars
import renderer from 'react-test-renderer';

import Info from '../../routes/info';

describe('routes/info.js', () => {
  const props = {
    navigation: {}
  };

  it('renders correctly', () => {
    const tree = renderer.create(
      <Info {...props} />
    );

    expect(tree).toMatchSnapshot();
  });
});
