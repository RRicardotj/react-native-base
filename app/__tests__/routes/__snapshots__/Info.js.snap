// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes/info.js renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 26,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    React Native Base
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 18,
        "marginVertical": 5,
        "textAlign": "center",
      }
    }
  >
    Add-On for React Native
  </Text>
</View>
`;
